src/docs/reference/modules/Users/README.md

```diff
@@ -22,7 +22,8 @@ If you want to specify custom paths to access the authentication related urls, y
       "LoginPath": "Login",
       "LogoffPath": "Users/LogOff",
       "ChangePasswordUrl": "ChangePassword",
-      "ExternalLoginsUrl": "ExternalLogins"
+      "ExternalLoginsUrl": "ExternalLogins",
+      "ChangePasswordConfirmationPath": "ChangePasswordConfirmation"
```

Comment:
```suggestion
      "ChangePasswordConfirmationUrl": "ChangePasswordConfirmation"
```

Can you move it up to below `ChangePasswordUrl`. 
Better that they match names, as you are introducing a new property.

Reply:
Alright. I’ll let you know when I’ll commit these changes.

Reply:
I committed the changes. You can see them clicking on the link below

https://github.com/OrchardCMS/OrchardCore/pull/10887/commits/ac50661586f63dcf5421fba83d79124151078b28